contracts/contracts/Sapphire.sol

```diff
@@ -1,28 +1,37 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.8;
+pragma solidity ^0.8.0;
```

Comment:
We wanted to standardize to `0.8.0`?

Reply:
I'm unaware of anything in earlier versions of Solidity 0.8 which would break the Sapphire contracts.

We should decide an explicit minimum version of Solidity to support, along with PNPM and NodeJS, and make it clear in the README and consistent across the files?

Reply:
➕ we should agree to minimum version and pin everything.